<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tablesDataV1</class>
 <widget class="QDialog" name="tablesDataV1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>968</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>BD JO</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>8</number>
     </property>
     <widget class="QWidget" name="tabLesEpreuves">
      <attribute name="title">
       <string>LesEpreuves</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QTableWidget" name="tableEpreuves">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>400</height>
          </size>
         </property>
         <property name="baseSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="sizeAdjustPolicy">
          <enum>QAbstractScrollArea::AdjustToContents</enum>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
         <property name="columnCount">
          <number>10</number>
         </property>
         <attribute name="horizontalHeaderCascadingSectionResizes">
          <bool>true</bool>
         </attribute>
         <attribute name="horizontalHeaderMinimumSectionSize">
          <number>50</number>
         </attribute>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>110</number>
         </attribute>
         <attribute name="horizontalHeaderHighlightSections">
          <bool>true</bool>
         </attribute>
         <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
          <bool>true</bool>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string>numEp</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>nomEp</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>forme</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>nomDi</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>categorieEp</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>nbSportifsEp</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>dateEp</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>numPOr</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>numPArgent</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>numPBronze</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_LesEpreuves">
         <property name="palette">
          <palette>
           <active>
            <colorrole role="WindowText">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>255</red>
               <green>0</green>
               <blue>0</blue>
              </color>
             </brush>
            </colorrole>
           </active>
           <inactive>
            <colorrole role="WindowText">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>255</red>
               <green>0</green>
               <blue>0</blue>
              </color>
             </brush>
            </colorrole>
           </inactive>
           <disabled>
            <colorrole role="WindowText">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>120</red>
               <green>120</green>
               <blue>120</blue>
              </color>
             </brush>
            </colorrole>
           </disabled>
          </palette>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabLesSportifs_base">
      <attribute name="title">
       <string>LesSportifs_base</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QTableWidget" name="tableSportifs">
         <attribute name="horizontalHeaderMinimumSectionSize">
          <number>50</number>
         </attribute>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>110</number>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string>numSp</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>nomSp</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>prenomSp</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>pays</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>categorieSp</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>dateNaisSp</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_LesSportifs_base">
         <property name="palette">
          <palette>
           <active>
            <colorrole role="WindowText">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>255</red>
               <green>0</green>
               <blue>0</blue>
              </color>
             </brush>
            </colorrole>
           </active>
           <inactive>
            <colorrole role="WindowText">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>255</red>
               <green>0</green>
               <blue>0</blue>
              </color>
             </brush>
            </colorrole>
           </inactive>
           <disabled>
            <colorrole role="WindowText">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>120</red>
               <green>120</green>
               <blue>120</blue>
              </color>
             </brush>
            </colorrole>
           </disabled>
          </palette>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabLesEquipes_base">
      <attribute name="title">
       <string>LesEquipes_base</string>
      </attribute>
      <widget class="QTableWidget" name="tableEquipes">
       <property name="geometry">
        <rect>
         <x>15</x>
         <y>11</y>
         <width>921</width>
         <height>471</height>
        </rect>
       </property>
       <column>
        <property name="text">
         <string>numEq</string>
        </property>
       </column>
      </widget>
      <widget class="QLabel" name="label_LesEquipes_base">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>490</y>
         <width>911</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tabLesMembresEquipes">
      <attribute name="title">
       <string>LesMembresEquipes</string>
      </attribute>
      <widget class="QTableWidget" name="tableMembresEquipes">
       <property name="geometry">
        <rect>
         <x>5</x>
         <y>11</y>
         <width>931</width>
         <height>471</height>
        </rect>
       </property>
       <column>
        <property name="text">
         <string>numEq</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>numSp</string>
        </property>
       </column>
      </widget>
      <widget class="QLabel" name="label_LesMembresEquipes">
       <property name="geometry">
        <rect>
         <x>6</x>
         <y>490</y>
         <width>931</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tabLesParticipants">
      <attribute name="title">
       <string>LesParticipants</string>
      </attribute>
      <widget class="QTableWidget" name="tableParticipants">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>921</width>
         <height>471</height>
        </rect>
       </property>
       <column>
        <property name="text">
         <string>numP</string>
        </property>
       </column>
      </widget>
      <widget class="QLabel" name="label_LesParticipants">
       <property name="geometry">
        <rect>
         <x>16</x>
         <y>490</y>
         <width>911</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tabLesDisciplines">
      <attribute name="title">
       <string>LesDisciplines</string>
      </attribute>
      <widget class="QTableWidget" name="tableDisciplines">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>921</width>
         <height>471</height>
        </rect>
       </property>
       <column>
        <property name="text">
         <string>nomDi</string>
        </property>
       </column>
      </widget>
      <widget class="QLabel" name="label_LesDisciplines">
       <property name="geometry">
        <rect>
         <x>16</x>
         <y>490</y>
         <width>911</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tabLesParticipations">
      <attribute name="title">
       <string>LesParticipations</string>
      </attribute>
      <widget class="QTableWidget" name="tableParticipations">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>931</width>
         <height>471</height>
        </rect>
       </property>
       <column>
        <property name="text">
         <string>numP</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>numEp</string>
        </property>
       </column>
      </widget>
      <widget class="QLabel" name="label_LesParticipations">
       <property name="geometry">
        <rect>
         <x>16</x>
         <y>490</y>
         <width>921</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tabLesSportifs">
      <attribute name="title">
       <string>LesSportifs</string>
      </attribute>
      <widget class="QTableWidget" name="tableSportifsView">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>0</y>
         <width>921</width>
         <height>481</height>
        </rect>
       </property>
       <column>
        <property name="text">
         <string>numSp</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>nomSp</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>prenomSp</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>pays</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>categorieSp</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>dateNaisSp</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>age</string>
        </property>
       </column>
      </widget>
      <widget class="QLabel" name="label_LesSportifs">
       <property name="geometry">
        <rect>
         <x>16</x>
         <y>490</y>
         <width>911</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tabLesEquipes">
      <attribute name="title">
       <string>LesEquipes</string>
      </attribute>
      <widget class="QTableWidget" name="tableEquipesView">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>921</width>
         <height>471</height>
        </rect>
       </property>
       <column>
        <property name="text">
         <string>numEq</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>nombreSp</string>
        </property>
       </column>
      </widget>
      <widget class="QLabel" name="label_LesEquipes">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>490</y>
         <width>911</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_data_V1">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_data_V1">
       <property name="text">
        <string>Fermer</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2_data_V1">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_data_V1</sender>
   <signal>clicked()</signal>
   <receiver>tablesDataV1</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>360</x>
     <y>575</y>
    </hint>
    <hint type="destinationlabel">
     <x>323</x>
     <y>169</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>refreshSpectaclesData()</slot>
  <slot>refreshZonesData()</slot>
  <slot>refreshRepresentationsData()</slot>
 </slots>
</ui>
